EXAMPLE.md

```diff
@@ -0,0 +1,302 @@
+# Running Tests in an openSUSE VM
+
+This is an example of setting up an [openQA][openqa] server with this set of
+tests and a worker inside an openSUSE QEMU VM.
+
+## Software Versions
+
+* openSUSE: `Leap 15.5`
+* openQA: `4.6.1667810206.2bf912d`
+
+## Documentation
+
+Instructions are available at <http://open.qa/docs/>.  The first part called
+"openQA starter guide" gives a nice general overview of openQA.
+
+## openSUSE Installation
+
+Download network installer ISO (203 MiB) for openSUSE Leap 15.5 from
+[here][opensuse-download] ([ISO][netinst-iso]):
+```
+wget -O opensuse.iso https://download.opensuse.org/distribution/leap/15.5/iso/openSUSE-Leap-15.5-NET-x86_64-Media.iso
+```
+
+Create a disk of suitable size (QCOW2 format grows on demand, it won't actually
+reach 128 GiB unless you'll fill it up that much, but it will grow to around
+20-60 GiB):
+```
+qemu-img create -f qcow2 disk.qcow2 128G
+```
+
+Start QEMU to perform the installation (use the same command to run the VM
+later, possibly dropping `-cdrom opensuse.iso` part):
+```
+qemu-system-x86_64 -m 6G \
+                   -cdrom opensuse.iso \
+                   -machine q35,accel=kvm \
+                   -drive if=virtio,file=disk.qcow2,format=qcow2 \
+                   -cpu host \
+                   -smp `nproc` \
+                   -netdev user,id=mynet0,hostfwd=tcp:127.0.0.1:4151-:22,hostfwd=tcp:127.0.0.1:4150-:80 \
+                   -device virtio-net,netdev=mynet0
+```
+
+### Installation Process
+
+The installer is quite standard and easy to use, so won't go into much detail.
+
+1. Pick "Installation" in boot menu, booting and then starting the installer
+   will take some time as it downloads packages.
+2. Accept license, probably keep English locale to keep it simple.
+3. Agree to activate "Online Repositories", then press "Next" to actually do it
+   (default state of the list seems sensible).  Wait for a bit.
+4. Pick "Server" for system role as we don't need any desktop environment.
+5. Accept default partitioning (BTRFS with a bunch of subvolumes) or adjust to
+   something like: boot partition, EXT4 root and no swap.
+6. Pick a time zone.  Can use localtime to avoid time difference in Web-UI.
+7. Don't bother creating a regular user.
+8. Pick root's password.
+9. Installation summary:
+   * Can disable CPU Mitigations in "Security" section for slightly better
+     performance.
+   * SSH should already be enabled in "Security" section.
+10. Hit "Install" and come back in about 5 minutes.
+
+### First Boot
+
+Connect via SSH:
+```
+ssh -p 4151 root@127.0.0.1
+```
+
+Disable AppArmor right away (alternative is to configure it properly for the
```

Comment:
FWIW, our production workers keep AppArmor enabled, with this addition (as `/etc/apparmor.d/local/usr.share.openqa.script.worker`):
```
# Site-specific additions and overrides for 'usr.share.openqa.script.worker'

/proc/cpuinfo r,
/usr/bin/base64 rix,
/usr/bin/tar rix,
/usr/bin/bash rix,
/var/lib/openqa/share/tests/qubesos/utils/* rk,
/var/lib/openqa/share/factory/hdd/* rk,
/var/lib/openqa/share/factory/hdd/fixed/* rk,
/var/lib/openqa/share/factory/iso/* rk,
/var/lib/openqa/share/factory/iso/fixed/* rk,
/var/lib/openqa/share/factory/repo/** r,
```

Reply:
Thanks, added.  Just disabling AppArmor was easier and adding only things that I've noticed could result in a partial list, so I didn't go for it.